community/team/index.md

```diff
@@ -164,6 +164,25 @@ permalink: /community/team/index.html
           </div>
         </div>
       </div>
+      <div class="left half col-lg-6 col-md-6 col-sm-12 col-xs-12">
+        <div class="info-block">
+          <div class="row center-xs">
+            <h2>{% t team.mpwg %}</h2>
+          </div>
+          <h4>{% t team.descr %}</h4>
+          <p>{% t team.mpwg_descr %}</p>
+          <h4>{% t team.start %}</h4>
+          <p>{% t team.mpwg_start %}</p>
+          <h4>{% t team.contacts %}</h4>
+          <ul class="logo">
+            <li>{% t team.chat %} <code>#monero-policy</code> <a class="chats-img" href="irc://chat.freenode.net/#monero-policy"><img class="freenode" src="/img/freenode.png" title="freenode" alt="freenode logo"></a> <a class="chats-img" href="https://matrix.to/#/#freenode_#monero-policy:matrix.org"><img class="matrix" src="/img/matrix-logo.svg" title="Matrix" alt="Matrix logo"></a></li>
+            <li>Email: <code>policy[at]getmonero[dot]org</code></li>
+          </ul>
+          <div class="row center-xs icons">
+            <a class="ext-noicon" href="https://github.com/monero-project/" target="_blank" rel="noreferrer, noopener" aria-label="GitHub logo"><div class="col social-icon github"></div></a>
```

Comment:
Please remove the GitHub logo (and this whole div), since the workgroup doesn't have a GitHub repository